scrapers/LoveHerFeet.yml

```diff
@@ -22,7 +23,7 @@ xPathScrapers:
         postProcess:
           - parseDate: January 2, 2006
       Image:
-        selector: //base/@href|//video/@poster
+        selector: //div[@class='video']//img[@class='mainThumb']/@src0_3x or (//base/@href|//video/@poster)
```

Comment:
`or` doesn't work like that
The above selector with the concat and the postprocessing doesnt work for all cases (BTS and plain scenes) 
Also the img class in she loves black BTS is a little different)
I had to use something like 
```yaml
      Image:
        selector: //div[@class='video']//img[contains(@class,'mainThumb')]/@src0_3x|//video/@poster|//base/@href
        concat: "|"
        postProcess:
          - replace:
              - regex: ^(https[^|]+)\|.+ # mainThumb urls do not need anything
                with: $1
              - regex: ^/tour([^|]+)\|(.+) # video/@poster urls need a domain
                with: $2$1
```
The pasted Image part seems to work fine, can you verify for scenes and BTS videos from all sites in the scraper?

Reply:
thanks for making sense of what i was aiming for.